tracing-error/src/heap_error.rs

```diff
@@ -59,12 +59,14 @@ impl Error for ErrorImpl {
 
 impl Debug for ErrorImpl {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        writeln!(f, "Instrumented Error SpanTrace:")?;
```

Comment:
why not just:
```suggestion
        writeln!(f, "span backtrace:")?;
```
as we printed previously?